tests/test_fastapi.sh

```diff
@@ -21,5 +21,14 @@ cd .. && pip install . && cd fastapi
 # To skip a specific test, add '--deselect path/to/test.py::test_name' to the end of this command
 #
 # To update the list of deselected tests, remove all deselections, run the tests, and re-add any remaining failures
-# TODO remove this once that test is fixed, see https://github.com/pydantic/pydantic/pull/10029
-./scripts/test.sh -vv --deselect tests/test_openapi_examples.py::test_openapi_schema
+
+# Remove the first one once that test is fixed, see https://github.com/pydantic/pydantic/pull/10029
+# the remaining tests all failing bc we now correctly add a `'deprecated': True` attribute to the JSON schema,
+# So it's the FastAPI tests that need to be updated here
+./scripts/test.sh -vv \
```

Comment:
FYI I think the first one is already passingm and the others should be fixed by https://github.com/fastapi/fastapi/pull/12971/files.

Reply:
I'd be happy to! Judging by my past experience with FastAPI I wouldn't expect that to happen any time soon 🙃

Reply:
https://github.com/pydantic/pydantic/pull/10948